client/components/status/status.test.js

```diff
@@ -464,6 +434,53 @@ describe("<Status /> interactions", () => {
       "4e:ed:11:2b:17:ae",
     );
   });
+  it("should not submit loginFormRef if user is already authenticated", async () => {
```

Comment:
why didn't you update this test description too? Please avoid the need of me telling you everything, If I ask you to make descriptions more explicit in one place, I'm assuming you're smart enough to understand you have to maintain consistency and make it explicit everywhere. Make this description consistent with the one below.